src/Agoda.Analyzers/AgodaCustom/AG0019PreventUseOfInternalsVisibleToAttribute.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Agoda.Analyzers.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+
+namespace Agoda.Analyzers.AgodaCustom
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class AG0019PreventUseOfInternalsVisibleToAttribute : DiagnosticAnalyzer
+    {
+        public const string DiagnosticId = "AG0019";
```

Comment:
I know it's silly, but the standards for constants are `DIAGNOSTIC_ID`

Reply:
Haha. I use `DIAGNOSTIC_ID` first then i saw other rules use `DiagnosticId` then i changed it. Agree with you It should `DIAGNOSTIC_ID`